h2o-r/h2o-package/R/stackedensemble.R

```diff
@@ -28,6 +28,7 @@
 #'        classification problems. Must be one of: "AUTO", "Random", "Modulo", "Stratified".
 #' @param metalearner_fold_column Column with cross-validation fold index assignment per observation for cross-validation of the metalearner.
 #' @param keep_levelone_frame \code{Logical}. Keep level one frame used for metalearner training. Defaults to FALSE.
+#' @param metalearner_params Parameters for metalearner algo Defaults to .
```

Comment:
This should say "Defaults to NULL" instead of "Defaults to ."   

I see that some params note their defaults and some do not (i think this is controlled both via the schema/api on the backend and the `gen_R.py` file, or it might just be the latter).

Reply:
Might be a typo in the auto-gen. Will check